src/loaders/genome_collection/parse_tool_results.py

```diff
@@ -839,7 +839,7 @@ def gtdb_tk(root_dir, kbase_collection, load_ver, fatal_ids):
     result_dir = _locate_dir(root_dir, kbase_collection, load_ver, tool='gtdb_tk')
     batch_dirs = _get_batch_dirs(result_dir)
 
-    genome_id_col = loader_common_names.GENOME_ID_COL
+    genome_id_col = loader_common_names.GTDB_GENOME_ID_COL
```

Comment:
Hmm... some of these look like further evidence that we should start writing tests as we're getting regressions